testsuite/forge/src/backend/k8s/swarm.rs

```diff
@@ -346,13 +445,17 @@ impl Swarm for K8sSwarm {
         "See fgi output for more information.".to_string()
     }
 
-    fn inject_chaos(&mut self, chaos: SwarmChaos) -> Result<()> {
+    async fn inject_chaos(&mut self, chaos: SwarmChaos) -> Result<()> {
         self.inject_swarm_chaos(&chaos)?;
         self.chaoses.insert(chaos);
+        self.ensure_chaos_experiments_active().await?;
```

Comment:
I wonder if we can pass the `chaos` to `ensure_chaos_experiments_active()` and build the Api by matching on the chaos and checking if it injected. 

Alternatively, we could also make sure all chaoses are properly injected in the end, so we can ensure that one chaos did not make the other one fail possibly (which has happened in the past :) )

Reply:
Updated this to list both network and stress chaoses